docs/admin/index.rst

```diff
@@ -1,10 +1,9 @@
-.. _handbook:
-.. _handbooks:
-.. _user-handbooks:
```

Comment:
just double checking: We do not reference this anywhere under these names already, right?

Reply:
I don't think so, because I've just introduced that recently. Otherwise, referenced projects will tell us on CI.